src/CommandLine/Parser.cs

```diff
@@ -183,20 +183,20 @@ public void Dispose()
                     settings.EnableDashDash)(arguments, optionSpecs);
         }
 
-        private static ParserResult<T> MakeParserResult<T>(ParserResult<T> parserResult, ParserSettings settings)
+        private /*static*/ ParserResult<T> MakeParserResult<T>(ParserResult<T> parserResult, ParserSettings settings)

```

Comment:
It would be better to just remove the static modifier altogether.  We can see the change in diffs.